app/views/elements.nunjucks

```diff
@@ -1,76 +1,78 @@
 {% extends 'layout.nunjucks' %}
 
 {% block content %}
-  <h1>Heading one</h1>
-  <h2>Heading two</h2>
-  <h3>Heading three</h3>
-  <h4>Heading four</h4>
-  <h5>Heading five</h5>
-  <h6>Heading six</h6>
+  <main id="content" role="main" tabindex="-1">
```

Comment:
It's more just that it's a common part of the layout now that the skiplinks have been added. So they're effectively tied together. 

If we need to add it to every view it makes it easier to forget and miss out which means the skiplinks in the main layout then become broken.